RELEASE-NOTES.md

```diff
@@ -1,5 +1,67 @@
 # Prefect Release Notes
 
+## Release 2.16.5
+
+### Exciting New Features 🎉
+- Render artifacts and states on the flow run graph by default. — https://github.com/PrefectHQ/prefect/pull/12300
+
+### Enhancements
+- Keep artifacts file in prefect-client — https://github.com/PrefectHQ/prefect/pull/12316
+- Add a select all flow runs checkbox to the flow runs page. — https://github.com/PrefectHQ/prefect/pull/12356
+- remove feature flagging around enhanced-deployment-experiment — https://github.com/PrefectHQ/prefect/pull/12360
+- Feature : #11773 UI: Add checkboxes for runs for an individual flow to allow multi-selection/-deletion — https://github.com/PrefectHQ/prefect/pull/12285
+
+### Fixes
+- Refactor logic to set `task_key` for background tasks — https://github.com/PrefectHQ/prefect/pull/12337
+- Correct a memory leak with the outbound task run websockets — https://github.com/PrefectHQ/prefect/pull/12346
+
+### Experimental
+- Create `CsrfToken` model and utilities — https://github.com/PrefectHQ/prefect/pull/12289
+- Create `csrf_token` endpoint to generate tokens for clients — https://github.com/PrefectHQ/prefect/pull/12297
+- Integrate `CsrfMiddleware` into server — https://github.com/PrefectHQ/prefect/pull/12303
+- Add CSRF support to orchestration client — https://github.com/PrefectHQ/prefect/pull/12314
```

Comment:
maybe these should be like Prefect Server and Prefect client

Reply:
Hmm, yeah. "Prefect Server" technically isn't a thing in 2.0, or it is, depending on who you ask. But maybe I can live in both realities with the right verbiage.

Reply:
ah yeah, true true. I think we say "Prefect server" now in the docs

Reply:
e.g. "a Prefect server"